deep_qa/contrib/background_search/bow_lsh.py

```diff
@@ -0,0 +1,170 @@
+'''
+The retrieval method encodes both the background and the query sentences by averaging their, and does retrieval using
+a Locality Sensitive Hash (LSH). We use ScikitLearn's LSH.
+'''
+import sys
+import os
+import argparse
+import gzip
+import numpy
+import sklearn
+import pickle
+
+import spacy
+from sklearn.neighbors import LSHForest
+
+
+class BOWLSH:
```

Comment:
Using a camel case here would be more readable: `BowLsh`.  A class docstring briefly describing what this class does would also be nice.